spec/controllers/users_controller_spec.rb

```diff
@@ -77,6 +77,13 @@
       expect(user.username).to eq("user234")
       expect(response).to redirect_to facility_user_path(facility, user)
     end
+
+    it_should_allow(:admin, "to log the change of the internal flag and log the new rate as metadata") do
```

Comment:
Can we also include a test that specifies that it does not log if the price group does not change?

Also, it's possible we have the feature setting overridden in the spec, but can you make sure this test still works when the setting `user_based_price_groups` is set to false (as it is at Dartmouth)?